### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joyfulrabbit**
tiny changes, big impact

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -693,8 +693,8 @@ username = "your-email"
 password = "password-or-api-key-for-confluence-cloud"
 # If you are using Confluence Cloud add the /wiki suffix to base_url
 base-url = "http://confluence.local"
-h1-title = true
-h1-drop = true
+h1_title = true
```
Comment (mrueg) [inline README.md:696]:
```suggestion
title-from-h1 = true
```

File: README.md [hunk 2]
```diff
@@ -693,8 +693,8 @@ username = "your-email"
...
+h1_drop = true
```
Comment (mrueg) [inline README.md:697]:
```suggestion
drop-h1 = true
```

**Review by mrueg — COMMENTED** [review]
Good catch! Thanks, one request: Please use the new options with `-`

File: README.md (hunk 2 above)
Comment (joyfulrabbit) [reply README.md:697]: I don't know if it really matters but your suggestion is visually troublesome for me.  Changing the ordering, using hyphens and underscores intermixed, is inconsistent. The solution I offered works, and follows a pattern. 
Make the user experience better by following patterns that make the features intuitive.

**Comment by joyfulrabbit** [comment]
Ship it!

**Comment by mrueg** [comment]
Thanks!

**Pull request merged**